src/Microsoft.Graph.Core/Requests/BaseRequest.cs

```diff
@@ -340,47 +336,6 @@ internal string BuildQueryString()
             return null;
         }
 
-        /// <summary>
-        /// Adds all of the headers from the header collection to the request.
-        /// </summary>
-        /// <param name="request">The <see cref="HttpRequestMessage"/> representation of the request.</param>
-        private void AddHeadersToRequest(HttpRequestMessage request)
-        {
-            if (this.Headers != null)
-            {
-                foreach (var header in this.Headers)
-                {
-                    request.Headers.TryAddWithoutValidation(header.Name, header.Value);
-                }
-            }
-
-            if (string.IsNullOrEmpty(this.sdkVersionHeaderValue))
-            {
-                var assemblyVersion = this.GetType().GetTypeInfo().Assembly.GetName().Version;
-                this.sdkVersionHeaderValue = string.Format(
-                    CoreConstants.Headers.SdkVersionHeaderValueFormatString,
-                    this.SdkVersionHeaderPrefix,
```

Comment:
Where do we set the sdk version header?

Reply:
This will move into the Telemetry Handler once we implement that.